packages/plugin-support/test/integration/spec/support.js

```diff
@@ -0,0 +1,75 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import {defaults, includes} from 'lodash';
+import fh from '@ciscospark/test-helper-file';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+
+describe(`plugin-support`, function() {
+  this.timeout(20000);
+
+  let spark;
+
+  let sampleTextOne = `sample-text-one.txt`;
+
+  before(() => Promise.all([
+    fh.fetch(sampleTextOne)
+  ])
+    .then((res) => {
+      [
+        sampleTextOne
+      ] = res;
+    }));
+
+  beforeEach(() => testUsers.create({count: 1})
+    .then((users) => {
+      spark = new CiscoSpark({
+        credentials: {
+          authorization: users[0].token
+        }
+      });
+    }));
+
+  describe(`#submitCallLogs()`, function() {
+    it(`uploads call logs for authUser`, function() {
+      return spark.support.submitCallLogs({}, sampleTextOne)
+        .then(function(body) {
+          assert.isDefined(body);
+          assert.isDefined(body.url);
+        });
+    });
+  });
+
+  describe(`#submitCallLogsForUnAuthUser()`, function() {
```

Comment:
this function no longer exists, so this test should be in submitCallLogs(). maybe add two separate describe blocks there, something like:

```javascript
describe(`#submitLogs()`, () => {
  describe(`when the current user is not authenticated`, () => {
    ...
  });

  describe(`when the current user is authenticated`, () => {
    ...
  });
});
```